src/main/kotlin/inc/evil/d17map/controllers/FloorController.kt

```diff
@@ -14,7 +14,7 @@ import org.springframework.web.bind.annotation.*
 
 
 @RestController
-@RequestMapping("/floors")
+@RequestMapping("/buildings/{buildingName}/floors")
```

Comment:
Dodaj tu też ten companion object (możesz nawet przekleić), to potem będzie łatwiejszy refactoring